types/adobe__pdfservices-node-sdk/operation/reorder-pages-operation.d.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * An operation that allows to rearrange pages in a PDF file according to the specified order.
+ *
+ * Sample Usage:
+ * <pre class="prettyprint">
+ * <code>
+ *  const credentials = PDFServicesSdk.Credentials.serviceAccountCredentialsBuilder()
+ *            .fromFile("pdfservices-api-credentials.json")
+ *            .build(),
+ *        executionContext = PDFServicesSdk.ExecutionContext.create(credentials),
+ *        ReorderPages = PDFServicesSdk.ReorderPages,
+ *        reorderPagesOperation = ReorderPages.Operation.createNew(),
+ *        pageRanges = new PDFServicesSdk.PageRanges();
+ *
+ *  reorderPagesOperation.setInput(PDFServicesSdk.FileRef.createFromLocalFile('~/Documents/reorderPagesOperationInput.pdf',
+ *  	ReorderPages.SupportedSourceFormat.pdf));
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/adobe__pdfservices-node-sdk/operation/reorder-pages-operation.d.ts;
  Line: 16;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.